docs/howtoguides/how_to_roll_out_across_multiple_machines.rst

```diff
@@ -0,0 +1,48 @@
+.. _roll_out_multiple_machines:
+
+How to roll out your pro token across multiple machines
+=======================================================
+
+This guide will show you how to roll out your pro token across multiple machines using cloud-init for configuration.
+
+Configure using Cloud-init
+--------------------------
+
+The next step is to usea cloud-init to configure your machines with the pro token.
+
+To do this, a cloud-init configuratioin file is required. Create a cloud-init.yaml file with the following content:
+
+.. code-block:: yaml
+
+    #cloud-config
+    ubuntu_pro:
+        token: <YOUR_PRO_TOKEN>
+
+
+Replace <YOUR_PRO_TOKEN> with the pro token you received from the Ubuntu Pro Dashboard.
```

Comment:
Obtaining the token from the dashboard should be step 1 - might need more detail here about that